### DISCUSSION TITLE
Small Latex font and latex not rendering

### DISCUSSION THREAD

**Discussion by FrancescoXD** [danny-avila/LibreChat#4973] [Troubleshooting]
### What happened?

Sometimes, it can happen that Gemini prints a weird latex or latex not rendered correctly (with $ at the start and at the end without rendering).
![Screenshot_20241212_001000_Chrome](https://github.com/user-attachments/assets/1e05d49b-050e-4fb0-8841-018256c134ad)

### Steps to Reproduce

1. Ask for a math resolution
2. Ask for katex or latex rendering

### What browsers are you seeing the problem on?

Chrome, Mobile (Android)

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
There's not enough information here to determine what the issue is exactly.

Can you provide a screen recording? The exact text that is being outputted? It may seem like it's a Gemini issue.

**Reply by FrancescoXD** [reply]
It is not a Gemini problem because in the AiStudio, if I say "Use LaTeX in $, no code blocks" it works perfectly.

The rendered LaTeX font is so small against the text font.

**Reply by FrancescoXD** [reply]
@danny-avila I can upload a screenshot. I was using Gemini Flash Thinking, and in the answer, I got:
![Screenshot_2025-01-11-17-32-47-683.jpeg](https://github.com/user-attachments/assets/1e1a0f5e-2a4c-4775-b5b9-433d7c378dd7)

You can see that the LaTeX font is weird compared to the normal text. It is not like ChatGpt interface or what is used to render on the AIStudio website.

**Reply by acgacgacg** [reply]
> It is not a Gemini problem because in the AiStudio, if I say "Use LaTeX in $, no code blocks" it works perfectly.
> 
> The rendered LaTeX font is so small against the text font.

Hi it worked for me! thank u

**Comment by hhh2210** [comment]
I have the same problem
![1925482062449205759](https://github.com/user-attachments/assets/29575898-d4d3-40c6-9614-f54da0e4d9b5)

**Reply by hhh2210** [reply]
I've already applied to this solution, not work https://github.com/danny-avila/LibreChat/blob/ea5140ff0fa53d8659ec4892b1cd2b2dc5aaf942/client/src/utils/latex.ts#L50

**Reply by danny-avila** [reply]
instruct the LLM to use LaTeX delimiter notation, e.g.:

![firefox_Slo4SLIFJL](https://github.com/user-attachments/assets/699f043b-8897-4f3e-a331-9f04660a8300)

**Reply by danny-avila** [reply]
you can put this into Agent instructions if you rely on LaTeX a lot.
